docs/getting_started/typescript/hello_world_in_typescript/index.md

```diff
@@ -234,316 +321,409 @@ async function run() {
-2023-11-22T04:38:13.298Z [INFO] runtime modules 937 bytes 4 modules
-2023-11-22T04:38:13.298Z [INFO] modules by path ./node_modules/@temporalio/ 209 KiB
-2023-11-22T04:38:13.298Z [INFO]   modules by path ./node_modules/@temporalio/common/ 111 KiB 19 modules
-2023-11-22T04:38:13.298Z [INFO]   modules by path ./node_modules/@temporalio/workflow/ 95.2 KiB
-2023-11-22T04:38:13.298Z [INFO]     ./node_modules/@temporalio/workflow/lib/worker-interface.js 11.8 KiB [built] [code generated]
-2023-11-22T04:38:13.299Z [INFO]     ./node_modules/@temporalio/workflow/lib/alea.js 2.87 KiB [built] [code generated]
-2023-11-22T04:38:13.299Z [INFO]     + 11 modules
-2023-11-22T04:38:13.299Z [INFO]   ./node_modules/@temporalio/worker/lib/workflow-log-interceptor.js 2.25 KiB [built] [code generated]
-2023-11-22T04:38:13.299Z [INFO] modules by path ./src/ 784 bytes
-2023-11-22T04:38:13.299Z [INFO]   ./src/workflows-autogenerated-entrypoint.js 540 bytes [built] [code generated]
-2023-11-22T04:38:13.299Z [INFO]   ./src/workflows.ts 244 bytes [built] [code generated]
-2023-11-22T04:38:13.299Z [INFO] __temporal_custom_payload_converter (ignored) 15 bytes [built] [code generated]
-2023-11-22T04:38:13.299Z [INFO] __temporal_custom_failure_converter (ignored) 15 bytes [built] [code generated]
-2023-11-22T04:38:13.299Z [INFO] ./node_modules/ms/index.js 2.95 KiB [built] [code generated]
-2023-11-22T04:38:13.299Z [INFO] webpack 5.75.0 compiled successfully in 540 ms
-2023-11-22T04:38:13.303Z [INFO] Workflow bundle created { size: '0.68MB' }
-2023-11-22T04:38:13.751Z [INFO] Worker state changed { state: 'RUNNING' }
-```
-
-In the output, you will see the Worker options and their values, the Webpack bundling output, and timestamps from Nodemon describing which programs have been invoked.
-
-Now that you have your Worker running, it's time for you to start a Workflow Execution.
-
-## Configure a Workflow Execution
+...
 
-You can start a Workflow Execution by using the Temporal CLI or by writing code using the Temporal SDK.
+2024-12-09T18:27:39.412Z [INFO] Workflow bundle created { sdkComponent: 'worker', taskQueue: 'ip-address-ts', size: '0.82MB' }
+2024-12-09T18:27:39.493Z [INFO] Worker state changed {
+  sdkComponent: 'worker',
+  taskQueue: 'ip-address-ts',
+  state: 'RUNNING'
+}
+```
 
-Starting a Workflow Execution using the Temporal SDK involves connecting to the Temporal Server, configuring the Task Queue the Workflow should use, and starting the Workflow with the input parameters it expects. In a real application, you may invoke this code when someone submits a form, presses a button, or visits a certain URL. In this tutorial, you will create a `client.ts` file that triggers your Temporal Workflow.
+In the output, you see the Worker options and their values, the Webpack bundling output, and the state of the Worker.
+
+Before running this Workflow, write tests to prove it works as expected.
```

Comment:
```suggestion
Your Worker is running and is polling the Temporal Service for Workflows to run, but before you start your Workflow, you'll write tests to prove it works as expected.
```

Reply:
You mean "polling the Task Queue" to be a bit more specific?